eo-runtime/src/main/java/org/eolang/ExprReduce.java

```diff
@@ -135,4 +133,76 @@ public Phi get(final Phi rho) {
         }
         return new Data.ToPhi(acc);
     }
+
+    /**
+     * Builds a class that stores some attributes of ExprReduce object.
+     * <p/>Definition example:
+     * <code><pre>
+     *     final VarargExpr&lt;Long&gt; st = new Storage<>(
+     *         "plus",
+     *         "x",
+     *         Long.class
+     *     );
+     * </pre></code>
+     * @param <T> Type of arguments
+     * @since 1.0
+     */
+    public static final class Storage<T> {
```

Comment:
@potatmen It's definitely better that plain array. But `Strorage<T>` looks like DTO object. I exposes no behaviour but rather just a placeholder for data. 
The idea I suggested earlier was to extract some of the functionality (behaviour) from `ExprReduce.get()` method.
Can you please think how we can redesign it?
As a start I can see that it does at least two things: (1) varargs extraction & validation and (2) reduction operation.

Reply:
@mximp so do you mean by this that I need just to create new methods in `Storage` class and move some functionality of `ExprReduce.get()` to these methods?

Reply:
@potatmen yes. Also I believe `Storage` is not quite correct name. Can you think of a name which would identify what object represent.